index.html

```diff
@@ -39,7 +43,215 @@ <h1>I'm Alishah. Glad to see you!</h1>
+                        bulid a product, feature or website.<br>
+                        Look through some of my work and <br>
+                        experience! if you like what you see and <br>
+                        have a project you need coded, don't <br>
+                        hesitate to contact me.
+                    </p>
+                </div>
+
+                <div class="social-media1">
+                    <h3 class="sub-social-media ">LET’S CONNECT</h3>
+
+                    <ul class="socail-media-icon1">
+                        <li> <a href="#"><img src="./Image/Vector (1).png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Linkedin icon.png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector (4).png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector.png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector (3).png" alt=""></a></li>
+                    </ul>
+                </div>
+                <div>
+                    <button class="normal-button">Get my resume</button>
+                </div>
+            </div>
+            <div class="list-skills">
+                <ul class="ul-item">
+                    <li>
+                        <ul class="language">
+                            <li>Languages</li>
+                            <li><img src="Image/Disabled.png" alt="bisabled sign-ups"></li>
+
+                        </ul>
+                        <ul>
+                            <li class="web-lang">
+                                <img src="Image/Ellipse 1.png" alt="bisabled sign-ups">
+
+                                <span>JavaScript</span>
+                            </li>
+                            <li class="web-lang">
+                                <img src="Image/Ellipse 1 (1).png" alt="bisabled sign-ups">
+                                <span>HTML</span>
+                            </li>
+                            <li class="web-lang">
+                                <img src="Image/Ellipse 1 (2).png" alt="bisabled sign-ups">
+                                <span>CSS</span>
+                            </li>
+
+                        </ul>
+                    </li>
+
+                    <li>
+                        <ul class="language">
+                            <li>Frameworks</li>
+                            <li><img src="Image/list.png" alt="bisabled sign-ups"></li>
+                        </ul>
+                        <hr>
+                    </li>
+                    <li>
+                        <ul class="language">
+                            <li>Skills</li>
+                            <li><img src="Image/list.png" alt="bisabled sign-ups"></li>
+                        </ul>
+                        <hr>
+                    </li>
+                </ul>
```

Comment:
- [ ] As requested by the last reviewer, Kindly consider using a nested unordered list to markup your list of technologies. Note that 'languages', 'frameworks', and 'skills' are list items of the parent unordered list and 'html', 'css', and 'javascirpt' are list items under languages. THis is better for HTML semantics. Kindly use the markup below as a guide;
```html

<ul>
  <li>Languages
    <ul>
        <li>JavaScript</li>
        <li>HTML</li>
        <li>CSS</li>
    </ul>
  </li>
  <li>Frameworks
    <ul>
        <li>React</li>
        <li>Ruby on Rails</li>
        <li>Rspec</li>
    </ul>
  </li>
  <li>Skills
    <ul>
        <li>Version control</li>
    ...
    </ul>
  </li>
</ul>
```